src/include/migraphx/check_shapes.hpp

```diff
@@ -238,6 +238,16 @@ struct check_shapes
         return *this;
     }
 
+    /*!
+     * Check all shapes are scalar.
+     */
+    const check_shapes& scalar() const
```

Comment:
Add unit-test for this ? We have `test/check_shapes.cpp` but not many unit-tests inside them.

Reply:
removing this addition